Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -163,6 +163,9 @@ Supporting infrastructure and systems MAY include:
 - backup
 - resources to support redundancy
 - resources to support failover
+- End user devices
```

Comment:
@bninassi I would recommend adding a few examples to each of these 3 categories to make them more useful to the reader of the SCI.

Reply:
Intended to be inclusive, overlap is ok, no need to include examples.